main.js

```diff
@@ -80,7 +80,7 @@ function draw () {
     fill(255);
     textSize(26);
     text('Attempts: ' + gameState.attempts, 600, 125);
-    text('Matches: ' + gameState.numMatches, 600, 150);
+    text('Matches: ' + gameState.numMatched, 600, 150);
```

Comment:
You had numMatches and numMatched in different places so I went with one (numMatched)